mne/datasets/utils.py

```diff
@@ -43,15 +45,14 @@
         it will not be downloaded and the path will be returned as
         '' (empty string). This is mostly used for debugging purposes
         and can be safely ignored by most users.
-    verbose : bool, str, int, or None
-        If not None, override default verbose level (see :func:`mne.verbose`).
+    %(verbose)s
 
     Returns
     -------
     path : str
         Path to {name} dataset directory.
 """
-
+_data_path_doc = docformat(_data_path_doc, docdict)
```

Comment:
why do you need sometimes to do more than just updating the docstring?

Reply:
All this does is update the docstring, but it can't be done with a decorator (*very* special / only case in our code) because we construct a special string here, and update/fill it later. Eventually this could probably be `doccer`-ized, but not as part of this PR (and maybe never really worth it).